spec/services/collection_change_set_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CollectionChangeSet do
+  let(:collection1) { create(:collection) }
+  let(:collection2) do
+    create(:collection, email_when_participants_changed: true, email_depositors_status_changed: true,
+                        review_enabled: true)
+  end
+
+  describe '#email_depositors_status_changed_changed?' do
```

Comment:
changed_changed?

Reply:
Unfortunately. The field is called `email_depositors_status_changed` so checking whether this field has changed appends `_changed?`.